### PR TITLE
Ja scss

### PR DESCRIPTION
**Pull request by adamsjr8576**
@Jessmaxim303 @Dustin-Mikusko @khalidwilliams @brittanystoroz

What's this PR do?
 - This PR converts the styles.css file to implement sass creating a new styles, variables, mixins and normalize files for scss. Refactor css for DRY code and increase general font-size and color contrast to increase accessibility. Also, refactor HTML for DRY code and eliminate unused classes and add id's for jQuery selecting instead of classes.
 - added a svg for the friends card.
 - utlizes scss variables and @extend
 - changes blue to darker blue and add more yellow for styling

Where should the reviewer start?
 - The viewer should start by looking at the new styles.scss file, which contains the majority of the changes - There is a good bit of reference to the variables.scss file as well - then look at HTML to check classes and id's to see how they connect to script.js file.

How should this be manually tested?
 - You could pull down and make sure that all looks good on your side.

Any background context you want to provide?
 - I've gotten good feedback on past projects that it is best practice to use classes for styling and id's for js selecting, which is why I made that decision and it also makes the HTML drier. The accessibility score is up to 91 with these changes!

What are the relevant tickets? #6 - #3 - #2

Questions:
 -  I think we should still split up the activity card into a couple smaller ones for easier use? 
 - Does this add new dependencies which need to be added? 
 - I checked and it looks like scss is already added as a dependency so we should be all good there.
 - It looks like the masonry layout css code might have gotten broken somehow, but I am not totally sure what is going on there. Once we figure out how to link packery to the webpack then that will hopefully solve the issues.
 
Screenshot:
screencapture-localhost-8080-2019-10-20-21_51_49

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,150 +1,139 @@
...
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600,800,900" rel="stylesheet" type="text/css">
+  <!-- <link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600,800,900" rel="stylesheet" type="text/css"> -->
   <script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
   <script src="https://cdn.rawgit.com/kimmobrunfeldt/progressbar.js/0.7.0/dist/progressbar.js"></script>
-
 </head>
-
 <body>
-  <section class="welcome-section">
```
Comment (Dustin-Mikusko) [inline src/index.html]: Nice work recognizing that this class isn't ever used in an effective way. This makes the HTML much more semantic.

File: src/css/variables.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+$font_weight_medium: 400;
+$font_weight_large: 600;
+$font_weight_small: 200;
+$font_size_xlarge: 3em;
+$font_size_large: 2em;
+$font_size_medium: 1.5em;
+$font_size_small: 1em;
+$text_align: center;
+$border: 3px solid #F1C40F ;
+$border_header: 3px solid #039BE5;
```
Comment (Dustin-Mikusko) [inline src/css/variables.scss]: Thank you for setting up the SASS file. It looks really good and is pretty neat!

Comment (Jessmaxim303) [reply src/css/variables.scss]: Everything looks great John. Im going to merge.

File: src/index.html [hunk 3]
```diff
@@ -1,150 +1,139 @@
...
-
+<html lang="en">
 <head>
   <title>Activity Tracker</title>
   <!-- <link rel="stylesheet" href="normalize.css"> -->
-  <link rel="stylesheet" href="styles.css">
+  <!-- <link rel="stylesheet" href="styles.scss"> -->
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600,800,900" rel="stylesheet" type="text/css">
+  <!-- <link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600,800,900" rel="stylesheet" type="text/css"> -->
```
Comment (khalidwilliams) [inline src/index.html]: Try and avoid committing commented out code to master

File: src/index.html [hunk 4]
```diff
@@ -1,150 +1,139 @@
...
-      <article class="data-article number-of-minutes-active-day-article">
+      <article class="data-article">
         <img class="img-medium" src="./images/stopwatch.svg" alt="road">
         <p class="subtext">TODAY</p>
-        <div class="separator">
-          <div class="user">
-            <h5>User</h5>
-            <h4 class="data-text number-of-minutes-active-day"></h4>
-            <p>Minutes Active</p>
+        <div class="side-by-side-container">
```
Comment (khalidwilliams) [inline src/index.html]: Nice job modifying the class names to more specific values

File: src/scripts.js [hunk 5]
```diff
@@ -65,20 +66,19 @@ function dropYear(dates) {
...
-
+  //
   // let $draggable = $('.draggable').draggabilly({
   //   containment: true
   // });
-
+  //
   // $grid.find('.grid-item').each(function (i, gridItem) {
   //   let draggie = new Draggabilly(gridItem)
   //   $grid.packery('bindDraggabillyEvents', draggie)
```
Comment (khalidwilliams) [inline src/scripts.js]: Same comment as above re: commented out code

File: dist/index.html [hunk 6]
```diff
@@ -1,152 +0,0 @@
...
-    </section>
-  </div>
-<script type="text/javascript" src="main.bundle.js"></script></body>
-<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.0/jquery.min.js"></script>
-<script src="https://cdn.rawgit.com/kimmobrunfeldt/progressbar.js/0.7.0/dist/progressbar.js"></script>
-<script src="https://unpkg.com/packery@2/dist/packery.pkgd.js"></script>
-<script src="https://unpkg.com/draggabilly@2/dist/draggabilly.pkgd.min.js"></script>
-
-</html>
```
Comment (khalidwilliams) [inline dist/index.html]: Make sure to add `/dist/` directory to your `.gitignore`
